src/problems/RadMarshakHot/test_radiation_marshak_hot.cpp

```diff
@@ -0,0 +1,265 @@
+//==============================================================================
+// TwoMomentRad - a radiation transport library for patch-based AMR codes
+// Copyright 2020 Benjamin Wibking.
+// Released under the MIT license. See LICENSE file included in the GitHub repo.
+//==============================================================================
+/// \file test_radiation_marshak_hot.cpp
+/// \brief Defines a test Marshak wave problem with extremely high specific heat capacity in radiation.
```

Comment:
What is the motivation for this test problem?

Reply:
It was intended for demonstrating that the updated Newton iteration converges faster. It's also used as a base for future multigroup test with dust and photoelectric heating.